e2e/bitrise.yml

```diff
@@ -370,9 +408,6 @@ workflows:
         - configuration: $BITRISE_CONFIGURATION
         - distribution_method: $DISTRIBUTION_TYPE
         - log_formatter: xcodebuild
-    - deploy-to-bitrise-io:
```

Comment:
I don't think this is needed, there is a single Deploy step at the end of the generated workflow that runs all E2E tests.